fullscreen/api/element-request-fullscreen-svg-svg-manual.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<title>Element.requestFullscreen() for SVG svg element</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="../trusted-click.js"></script>
+<div id="log"></div>
+<svg></svg>
+<script>
+async_test(function(t)
+{
+    var svg = document.querySelector("svg");
+    assert_true(svg instanceof SVGSVGElement);
+    trusted_request(svg, document.body);
+    document.onfullscreenchange = t.step_func_done();
```

Comment:
Ah, the test actually never exits fullscreen, which I guess is annoying when testing manually. If it did it'd still not be necessary to clean up this event handler to avoid the extra call to `t.step_func_done()` though.